ucsm_apis/admin/syslog.py

```diff
@@ -0,0 +1,368 @@
+
+
+def syslog_local_console_disable(handle):
+    """
+    This method disables system logs on local console.
+
+    Args:
+        handle (UcsHandle)
+
+    Returns:
+        CommSyslogConsole: Managed Object
+
+    Raises:
+        UcsOperationError: If CommSyslogConsole is not present
+
+    Example:
+        syslog_local_console_enable(handle)
+    """
+
+    from ucsmsdk.mometa.comm.CommSyslogConsole import \
+        CommSyslogConsoleConsts
+
+    dn = _base_dn + "/console"
+    mo = handle.query_dn(dn)
+    if not mo:
+        raise UcsOperationError("syslog_local_console_disable",
+                                 "syslog console does not exist.")
+
+    mo.admin_state = CommSyslogConsoleConsts.ADMIN_STATE_DISABLED
+    handle.set_mo(mo)
+    handle.commit()
+    return mo
+
+
+def syslog_local_monitor_enable(handle, severity="emergencies", **kwargs):
+    """
+    This method enables logs on local monitor.
+
+    Args:
+        handle (UcsHandle)
+        severity (string): Level of logging.
+                        ["alerts", "critical", "debugging", "emergencies",
+                        "errors", "information", "notifications", "warnings"]
+        **kwargs: Any additional key-value pair of managed object(MO)'s
+                  property and value, which are not part of regular args.
+                  This should be used for future version compatibility.
+    Returns:
+        CommSyslogMonitor: Managed Object
+
+    Raises:
+        UcsOperationError: If CommSyslogMonitor is not present
+
+    Example:
+        syslog_local_monitor_enable(handle, severity="alert")
+    """
+
+    from ucsmsdk.mometa.comm.CommSyslogMonitor import \
+        CommSyslogMonitorConsts
+
+    dn = _base_dn + "/monitor"
+    mo = handle.query_dn(dn)
+    if not mo:
+        raise UcsOperationError("syslog_local_monitor_enable",
+                                 "syslog monitor does not exist.")
+
+    mo.admin_state = CommSyslogMonitorConsts.ADMIN_STATE_ENABLED
+    mo.severity = severity
+
+    mo.set_prop_multiple(**kwargs)
+    handle.set_mo(mo)
+    handle.commit()
+    return mo
+
+
+def syslog_local_monitor_disable(handle):
+    """
+    This method disables logs on local monitor.
+
+    Args:
+        handle (UcsHandle)
+
+    Returns:
+        CommSyslogMonitor: Managed Object
+
+    Raises:
+        UcsOperationError: If CommSyslogMonitor is not present
+
+    Example:
+        mo = syslog_local_monitor_disable(handle)
+    """
+
+    from ucsmsdk.mometa.comm.CommSyslogMonitor import \
+        CommSyslogMonitorConsts
+
+    dn = _base_dn + "/monitor"
+    mo = handle.query_dn(dn)
+    if not mo:
+        raise UcsOperationError("syslog_local_monitor_disable",
+                                 "syslog monitor does not exist.")
+
+    mo.admin_state = CommSyslogMonitorConsts.ADMIN_STATE_DISABLED
+
+    handle.set_mo(mo)
+    handle.commit()
+    return mo
+
+
+def syslog_local_file_enable(handle, name=None, severity="emergencies",
```

Comment:
should `name` be a mandatory parameter ? Is a default name picked up when we don't specify one ?